src/Core/src/Platform/Android/TextAlignmentExtensions.cs

```diff
@@ -8,19 +8,27 @@ public static class TextAlignmentExtensions
 		internal static void UpdateHorizontalAlignment(this EditText view, TextAlignment alignment, AGravityFlags orMask = AGravityFlags.NoGravity)
 		{
 			if (!Rtl.IsSupported)
-				view.Gravity = alignment.ToHorizontalGravityFlags() | orMask;
+			{
+				// The mask is used to ignore the previous horizontal alignment settings
+				var horizontalGravityMask = AGravityFlags.CenterHorizontal | AGravityFlags.End | AGravityFlags.Start;
```

Comment:
Since these masks don't ever change, can we make them static members or constants?

Reply:
Totally. I'll make the change